src/main/kotlin/no/nb/mlt/wls/infrastructure/synq/SynqOrderPayload.kt

```diff
@@ -43,10 +43,16 @@ data class SynqOrderPayload(
 data class ShippingAddress(
     val address: Address
 ) {
+    // TODO - Should all the fields here be annotated?
```

Comment:
Yeah, I think the address as a whole and its fields should not be included if null

Reply:
The ShippingAddress is currently not nullable, due to `contactPerson` being a required field for the SynQ payload. Should this information be sent to storage? If not, then we can make it an optional field